src/api/controllers/lines.js

```diff
@@ -57,8 +57,9 @@ function createLine(req, res) {
             //otherwise this is fine
 
             var line = Line();
-            line.user_id = req.user._id
-            line.employer_id = req.body.employer_id
+            line.user_id = req.user._id;
+            line.employer_id = req.body.employer_id;
+            line.status = 'preline';
```

Comment:
if this is meant to force the created line to be `preline` status, then i guess this is fine. no changes needed.

Reply:
per discussion with john Wed 2/21, we should leave this as is (even though we're planning a skip ahead feature later this sprint).